Makefile

```
@@ -0,0 +1,55 @@
+.PHONY: build test lint vet generate clean install wasm help
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)